tests/HTMLConverterTest.php

```diff
@@ -54,6 +54,63 @@ public function testToHTML()
         self::assertContains('<table class="table-csv-data" id="test">', $html);
         self::assertContains('<tr data-record-offset="', $html);
         self::assertContains('<td title="', $html);
+        self::assertNotContains('<thead>', $html);
+        self::assertNotContains('<tbody>', $html);
+        self::assertNotContains('<tfoot>', $html);
+    }
+
+    /**
+     * @covers ::__construct
+     * @covers ::table
+     * @covers ::tr
+     * @covers ::td
+     * @covers ::convert
+     */
+    public function testToHTMLWithHeaders()
+    {
+        $csv = Reader::createFromPath(__DIR__.'/data/prenoms.csv', 'r')
+            ->setDelimiter(';')
+            ->setHeaderOffset(0)
+        ;
+
+        $stmt = (new Statement())
+            ->offset(3)
+            ->limit(5)
+        ;
+
+        $records = $stmt->process($csv);
+
+        $converter = (new HTMLConverter())
+            ->table('table-csv-data', 'test')
+            ->td('title')
+            ->tr('data-record-offset')
+        ;
+
+        $html = $converter->convert($records, $records->getHeader());
```

Comment:
if you only provide a `thead` why does the `tfoot` is present 🤔

Reply:
Honestly, I assumed that they came as a threesome...
I originally hid them accordingly, would you you prefer I did that again?

Reply:
yes hides them accordingly ... I prefer this to be explicit. I was unable to find a resource stating for instance that if you use `tfoot` then `thead` is required. The only requirement is that if one is present then `tbody` **must be used**.

Reply:
done